developer/src/kmc-model/test/wordbreakers/README.md

```diff
@@ -0,0 +1,3 @@
+Wordbreakers ES Module format
```

Comment:
The wordbreakers files in this folder are a copy of the files from common/models/wordbreakers/default, to avoid ESM yak shave with common .ts code before we are ready for it.